lambdatimes/src/components/Content/Tab.js

```diff
@@ -4,13 +4,17 @@ const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+      let selected = false;
+      if (props.tab === props.selectedTab) {
+        selected = true;
+      }
   return (
     <div
-      className={''}
-      onClick={() => {
+      className={selected ? 'tab active-tab' : 'tab'}//first time using this kind of cool
+      onClick={() =>  props.selectTabHandler(props.tab)}//this was a pain and I am not sure if there is a better way to do this without invoking the function
```

Comment:
this is the way you'd want to do it. Sorry if it was a headache to figure out but it's a common pattern in React for the future